src/libs/API.js

```diff
@@ -449,7 +453,9 @@ function DeleteLogin(parameters) {
     const commandName = 'DeleteLogin';
     requireParameters(['partnerUserID', 'partnerName', 'partnerPassword', 'shouldRetry'],
         parameters, commandName);
-    return Network.post(commandName, parameters);
+
+    // Note: Aborting request would happen while logging out, we want to skip abort the actual logout request
```

Comment:
This comment would be better if it explained "why" the logout request shouldn't be cancelled (same for `log()`)

Reply:
Oh, sorry I guess it's not clear

What I'm trying to say is that **this** DeleteLogin request is not abortable, because it is **the** logout request
During logout we abort all request but if we abort **this** one it would abort sending the `DeleteLogin` command to the backend

Can you help me convey that in a more understandable way?

Reply:
I think having the comment say exactly this would be perfect!

> DeleteLogin request is not abortable, because it is the logout request
During logout we abort all request but if we abort this one it would abort sending the DeleteLogin command to the backend

I love that explanation.

Reply:
Well it's seems you've liked this one as well
> Non-cancellable request: during logout, when requests are cancelled, we don't want to cancel the actual logout request

But I can swap it out if you really love the other one 😉

Reply:
haha, I do really love the newest one.